lagom/shopping-cart-scala/shopping-cart/src/main/resources/cinnamon.conf

```diff
@@ -22,32 +22,27 @@ cinnamon {
 
   chmetrics.reporters += "console-reporter"
 
+  #
+  # Uncomment if you use Prometheus as a backend.
+  # See https://developer.lightbend.com/docs/telemetry/current/sandbox/prometheus-sandbox.html
+  #
+  # prometheus.exporters += http-server
+
   akka.actors {
-    default-by-class {
-      includes = "/user/*"
-      report-by = class
-      excludes = ["akka.http.*", "akka.stream.*"]
+    "sharded-group" {
+      report-by = group
+      includes = ["/system/sharding/ShoppingCart/*"]
+      excludes = ["akka.cluster.sharding.Shard"]
     }
   }
-
-  lagom.http {
-    servers {
-      "*:*" {
-        paths {
-          "*" {
-            metrics = on
-          }
-        }
+  akka {
+    actors {
+      "com.lightbend.lagom.internal.scaladsl.persistence.ReadSideActor" {
+        report-by = group
       }
-    }
-    clients {
-      "*:*" {
-        paths {
-          "*" {
-            metrics = on
-          }
-        }
+      "com.lightbend.lagom.internal.broker.kafka.TopicProducerActor" {
+        report-by = group
       }
     }
```

Comment:
On second thought, I think we can use this sample to demonstrate some features of actor selection. So, I think we can keep these selectors referring to the docs you mentioned and commenting on the purpose of the selectors:

```

  #
  # In general, the settings recommended to produce metrics for  actors in Lagom applications should
  # be enough. As indicated in https://developer.lightbend.com/docs/telemetry/current/getting-started/lagom_java.html
  # these settings are:
  #
  #  akka.actors {
  #     default-by-class {
  #      includes = "/user/*"
  #      report-by = class
  #      excludes = ["akka.http.*", "akka.stream.*"]
  #    }
  #  }
  #
  # But you can also fine tune the actors telemetry to produce fewer data and report it in a more
  # meaningful format (e.g. grouping). For example, this sample application uses a single persistent
  # entity ("ShoppingCart") that's sharded. It also has two projections (a read side processor and 
  # a topic producer). So we can limit the actors telemetry to these types of actors:
  #
  akka.actors {
    "sharded-group" {
      report-by = group
      includes = ["/system/sharding/ShoppingCart/*"]
      excludes = ["akka.cluster.sharding.Shard"]
    }
    "com.lightbend.lagom.internal.scaladsl.persistence.ReadSideActor" {
      report-by = group
    }
    "com.lightbend.lagom.internal.broker.kafka.TopicProducerActor" {
      report-by = group
    }
  }
```

(GitHub suggesting is disabled because this chunk is either too big or has too many additions/removals)